storage/sql_hooks.go

```diff
@@ -83,9 +83,9 @@ func InitAndGetSQLDriverWithLogs(driverName string, logger *log.Logger) (string,
 	var driver sql_driver.Driver
 
 	switch driverName {
-	case "sqlite", "sqlite3":
+	case "sqlite", sqliteDriverName:
```

Comment:
maybe it's better to remove "sqlite" here?

Reply:
should it happen that "sqlite" will be set?

Reply:
so far we use sqlite3 in configs